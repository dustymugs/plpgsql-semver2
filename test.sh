#!/bin/bash

if [ -z "$POSTGRES_SEARCH_PATH" ]; then
	POSTGRES_SEARCH_PATH="public"
fi

for sql in $(find ./tests -name test.*.sql); do
	rm -rf /tmp/semver.out
	expected=${sql/%.sql/.expected}
	echo -n $sql "..."
	psql "options=--search_path=${POSTGRES_SEARCH_PATH}" -q -A -t -f "$sql"	> /tmp/semver.out
	diffs=$(diff /tmp/semver.out "$expected")
	if [ -z "$diffs" ]; then
		echo "OK"
	else
		echo $diffs
	fi
done

rm -rf /tmp/semver.out
