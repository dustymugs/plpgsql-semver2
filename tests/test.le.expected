f
t
t
f
t
t
t
t
t
t
t
t
t
t
t
